src/System.Text.Json/src/System/Text/Json/Serialization/JsonIgnoreAttribute.cs

```diff
@@ -10,6 +10,9 @@ namespace System.Text.Json.Serialization
     [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
     public sealed class JsonIgnoreAttribute : JsonAttribute
     {
+        /// <summary>
+        /// Initializes a new instance of <see cref="JsonIgnoreAttribute"/>.
```

Comment:
I suggest you usethe full DocId, like this:

```suggestion
        /// Initializes a new instance of <see cref="T:System.Text.Json.Serialization.JsonIgnoreAttribute"/>.
```

Reply:
As Stephen pointed out, references get resolved by the compiler to their full DocId.